solutions/cses-1736.mdx

```diff
@@ -0,0 +1,149 @@
+---
+id: cses-1736
+source: CSES
+title: Polynomial Queries
+author: Albert Ye
+---
+
+## Solution
+**Time Complexity: $\mathcal{O}(n\log n)$**
+
+In this question, we have range updates and range queries, but
+the updates are of a strange format: incrementing the first number
+in interval by $1$, the second by $2$, etc. However, despite the
+strange operations we can still use a segment tree with lazy propagation
+to solve this problem.
+
+The range queries should be similar to in other lazy problems: we finish
+the lazy propagation to all needed nodes and find the range query. However,
+the range updates are different. 
```

Comment:
Do we really need this?

Reply:
still worth saying that we are only going to focus on the updates in the sol